packages/wxt/src/core/resolve-config.ts

```diff
@@ -536,17 +536,10 @@ async function getUnimportEslintOptions(
  * Returns the path to `node_modules/wxt`.
  */
 function resolveWxtModuleDir() {
-  // TODO: Drop the __filename expression once we're fully running in ESM
-  // (see https://github.com/wxt-dev/wxt/issues/277)
```

Comment:
Read this issue a bit closer, is the reason we dynamically import vite to suppress the cjs warning? If so, we can switch to static imports now (there's already static imports in the codebase, so presumably it's just wasting cycles now).